src/Sylius/Component/User/Model/User.php

```diff
@@ -405,18 +405,22 @@ public function serialize(): string
             $this->enabled,
             $this->id,
             $this->encoderName,
-        ]);
+        ];
     }
 
     /**
-     * @param string $serialized
+     * @internal
      */
-    public function unserialize($serialized): void
+    public function serialize(): string
+    {
+        return serialize($this->__serialize());
```

Comment:
I think we should add a depreaction message

Reply:
Like this ?
```
/**
 * @internal
 * @deprecated since 1.11 and will be removed in Sylius 2.0, use \Sylius\Component\User\Model\User::__serialize() or \serialize($user) in PHP 8.1 instead
  */

Reply:
Yes :)